src/test/java/ci.sql

```diff
@@ -254,4 +254,32 @@ CREATE TABLE `test_ttl_timestamp` (
  `expired_ts` timestamp,
 PRIMARY KEY (`c1`)) TTL(expired_ts + INTERVAL 0 SECOND);
 
+CREATE TABLE IF NOT EXISTS `test_auto_increment_rowkey` (
+    `c1` int auto_increment,
+    `c2` int not null,
+    `c3` int default null,
+    primary key(c1, c2)
```

Comment:
I guess we should check the partition table.

Reply:
fixed